legal-api/src/legal_api/services/filings/validations/continuation_in.py

```diff
@@ -192,7 +192,7 @@ def validate_business_in_colin(filing_json: dict, filing_type: str) -> list:
         elif legal_name != response_json['business']['legalName']:
             msg.append({'error': 'Legal name does not match with company legal name from Colin.',
                         'path': business_legal_name_path})
-        elif founding_date != response_json['business']['founding_date']:
+        elif founding_date != response_json['business']['foundingDate']:
```

Comment:
Did the unit test catch this typo?

Reply:
Yes. I noticed this typo when I saw the failed unit test.